space_packet_parser/definitions.py

```diff
@@ -370,10 +370,8 @@ def _parse_header(packet_data: bytes) -> dict:
         header = {}
         current_bit = 0
         for item in CCSDS_HEADER_DEFINITION:
-            header[item.name] = packets.ParsedDataItem(
-                name=item.name,
-                # pylint: disable=protected-access
-                raw_value=packets._extract_bits(packet_data, current_bit, item.nbits))
+            # pylint: disable=protected-access
+            header[item.name] = packets._extract_bits(packet_data, current_bit, item.nbits)
```

Comment:
Remind me why `_extract_bits` is a private function?

Reply:
Do we want users to rely on this function? We can make it public if so, but I figured this was internal implementation details.

Reply:
I guess it's a difference of convention between private to the module vs private to the library. My understanding is that a _private_method is private to the module, which is why pylint is complaining that we're accessing a private function outside of the module in which it's defined.

I don't think users need it but the convention says we shouldn't be using it, even in a different module. But maybe I misunderstand that convention.

Reply:
I have the different take on it from pylint. The `_private()` is just to indicate it is something that users should not rely on and can be changed or removed at any point without a deprecation. This is the only place we use it outside of the module it is defined in, so maybe there is a better way to handle this, or maybe that is just to make this helper function public?

Reply:
I'm not stressed about it. LGTM